src/api/app/views/webui/request/build_results.html.haml

```diff
@@ -0,0 +1,37 @@
+:ruby
+  @pagetitle = "Request #{@bs_request.number}"
+  @pagetitle += ": #{@action[:name]}"
```

Comment:
Why those two lines are split?

Reply:
No specific reason. Other pages use the same style of initializing the title.